src/views/Package/components/ChooseSubmodule/ChooseSubmodule.tsx

```diff
@@ -43,18 +43,21 @@ export const ChooseSubmodule: FunctionComponent<ChooseSubmoduleProps> = ({
   );
 
   const submodules = useMemo(() => {
-    let results = assembly?.submodules ?? [];
+    let results = Object.keys(assembly?.submodules ?? {});
```

Comment:
Nit, but we could set a const in the component's scope for submodule names - 

`const submoduleNames  = Object.keys(assembly?.submodules ?? {});`

so that we can re-use this for the `disabled` prop in favor of repeating the `Object.keys` logic